### PR TITLE
#164856306 Update specific field on a question

### PR DESCRIPTION
**Pull request by Philipotieno**
#### What does this PR do?
- Has view function to edit specific field on questions
#### Description of Task to be completed?
- `PUT api/meetups/{meetup_id}/questions/{question_id}`
#### How should this be manually tested?
- `$ git clone https://github.com/Ogutu-Brian/questioners.git`
- `$ git fetch origin` - To get all branches
- `$ git checkout bg-update-question-164856306`
- Create a virtual environment and activate it
        `$ virtualenv -p python3 venv`
        ` $ source venv/bin/activate`
- Add a .env file as in the .env.example
        `$ source .env`
- ` $ pip intsall -r requirements.txt`
- Start server ` $ python api/manage.py runserver`
- Test the following endpoints on postman
   - Create superuser account `$ api/manage.py createsuperuser`
   - Sign in `/api/auth/signin/`
   - Create a meetup `/api/meetups/`
   - Post a question `/api/meetups/{meetup_id}/question`
   - Edit a question `PUT api/meetups/{meetup_id}/questions/{question_id}`
- Run tests ` $ python api/manage.py test`
#### What are the relevant pivotal tracker stories?
[164856306](https://www.pivotaltracker.com/n/projects/2315729/stories/164856306)
#### Screenshots
title update
title2
body1
body2

### CODE REVIEW

File: api/questions/views.py [hunk 1]
```diff
@@ -102,31 +102,41 @@ def put(self, request, id, m_id):
...
                 return Response(data=error, status=status.HTTP_404_NOT_FOUND)
             try:
-                request.data['created_by'] = request.user
-                is_authenticated = request.user
-                owner = Question.objects.filter(
-                    created_by_id=is_authenticated).first()
-                meetup_id = Question.objects.filter(meetup_id=m_id).first()
-
-                if owner:
+                qn = Question.objects.get(id=id)
```
Comment (Ogutu-Brian) [inline api/questions/views.py]: Hello @Philipotieno ,
A variable is a symbolic name for (or reference to) information. The variable's name represents what information the variable contains so it should be easy to notice it at a glance. 
You could consider updating the variable `qn` to an easy to notice variable name.
You can read [this](https://hackernoon.com/the-art-of-naming-variables-52f44de00aad) to learn more about such.

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]

**Review by Amoswachira — CHANGES_REQUESTED** [review]
work on @Ogutu-Brian's feedback

File: api/questions/views.py (hunk 1 above)
Comment (Philipotieno) [reply api/questions/views.py]: Fixing that right now @Ogutu-Brian

**Review by Amoswachira — APPROVED** [review]

**Comment by Amoswachira** [comment]
There are some merge conflicts

**Pull request merged**